cmd/create.go

```diff
@@ -27,6 +27,13 @@ var createCmd = &cobra.Command{
 	Short: "Provision an EKS cluster in AWS",
 	Long:  `Provision an EKS cluster in AWS.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+
+		// Create resource client
```

Comment:
Nit-picking here - but I think we should have two styles of comments:
* Complete sentences with grammar for comments that will be used in online docs
* Explanatory comments that begin with lower case and use incomplete sentences
The at-a-glance distinction is nice.  (I'll start a style guide that formalizes some of these things.)

Reply:
Are you thinking the former is used for function doc strings, and the latter used within functions? And should all explanatory comments within a function be explanatory?

Reply:
Yep - that's exactly what I'm thinking

Reply:
Sounds good, pushed these changes. I also left sentences beginning with a lowercase letter if they are referencing a variable that is lowercase, for example "// createCmd represents the create command." I think this alright, but let me know if you'd prefer otherwise.